attractions/date-picker/date-picker.svelte

```diff
@@ -25,6 +25,7 @@
   export let range = false;
   export let locale = undefined;
   export let firstWeekday = 1;  // 1 corresponds to Monday
+  export let noCalendar = false;
```

Comment:
We use double negation everywhere (`noRipple`, `noPadding`, `noPrefetch`) because passing those props looks more reasonable:
`<Component noSmth />`
as opposed to
`<Component smth={false} />`